documentation-site/components/example.js

```diff
@@ -107,24 +117,48 @@ class Example extends React.Component<PropsT, StateT> {
         {this.props.children}
 
         <Block paddingTop="scale400">
-          <Button
-            kind={KIND.secondary}
+          <ButtonGroup
             size={SIZE.compact}
-            onClick={() => {
-              this.setState(prevState => ({
-                isSourceOpen: !prevState.isSourceOpen,
-              }));
-              trackEvent('show_source', this.props.title);
+            selected={this.state.sourceSelected}
+            mode="radio"
+            onClick={(event, index) => {
+              if (this.state.sourceSelected !== index) {
+                this.setState({sourceSelected: index});
+              } else {
+                this.setState({sourceSelected: -1});
+              }
             }}
           >
-            {this.state.isSourceOpen ? 'Hide' : 'Show'} Source
-          </Button>
+            <Button
+              kind={KIND.secondary}
+              startEnhancer={() => <CodeIcon />}
+              onClick={() => {
+                trackEvent('show_js_source', this.props.title);
```

Comment:
is it worth maintaining the `show_source` event? i'm not sure if metrics are measured here cc @gergelyke

Reply:
I've added these metrics originally. I think we can just sum up `show_js_source` and `show_flow_source` if we ever want to compare it with `show_source`.

Reply:
would it be possible to always fire the show_source event, to make it easier?

Reply:
summing should be fine - i just wasn't aware if the data was being used

Reply:
> would it be possible to always fire the show_source event, to make it easier?

I am quite interested to see what's the favorite language of our users.

We could fire `show_source` in parallel but I felt it's not needed if we have a more fine-grained metric and we can sum it up.